### PR TITLE
Adds `useAcceptLanguages()` hook

### PR DESCRIPTION
**Pull request by cartogram**
## Description

Fixes (issue #)

## Type of change

- [ ]  Patch: Bug/ Documentation fix (non-breaking change which fixes an issue or adds documentation)
- [ ]  Minor: New feature (non-breaking change which adds functionality)
- [ ]  Major: Breaking change (fix or feature that would cause existing functionality to not work as expected)

## Checklist

- [ ] I have added a changelog entry, prefixed by the type of change noted above
- [ ] I have prefixed my pull request title with the corresponding emoji from [this guide](https://gitmoji.carloscuesta.me/)

### CODE REVIEW

**Comment by cartogram** [comment]
@lemonmade @TheMallen re-did this as a stand-alone hook in `react-network`, how does this look/ feel?

File: packages/react-network/src/hooks.ts [hunk 1]
```diff
@@ -38,3 +38,10 @@ export function useStatus(code: StatusCode) {
 export function useRedirect(url: string, status?: StatusCode) {
   useNetworkEffect(network => network.redirectTo(url, status));
 }
+
+export function useAcceptLanguages(fallback = 'en') {
+  const acceptsLanguages = useRequestHeader(Header.AcceptLanguage);
+  const locales = acceptsLanguages ? acceptsLanguages.split(',') : [fallback];
+
+  return [locales[0]];
```
Comment (lemonmade) [inline packages/react-network/src/hooks.ts]: Why return only the first one?

File: packages/react-network/src/test/hooks.test.tsx [hunk 2]
```diff
@@ -0,0 +1,62 @@
...
+
+  it('returns the locale from the language header', async () => {
+    const lang = 'it';
+    const wrapper = await mountWithLanguageHeader(<MockComponent />, lang);
+
+    expect(wrapper.find(MockComponent)).toContainReactText(lang);
+  });
+
+  it('returns the first locale from the language header with multiple languages', async () => {
+    const lang = 'it, fr, en-US';
```
Comment (lemonmade) [inline packages/react-network/src/test/hooks.test.tsx]: IIRC the actual header also has percentage matches that we'll need to strip out of the string (example from MDN: `fr-CH, fr;q=0.9, en;q=0.8, de;q=0.7, *;q=0.5`)

File: packages/react-network/src/test/hooks.test.tsx [hunk 3]
```diff
@@ -0,0 +1,62 @@
...
+  });
+
+  it('returns `en` if no fallback is set and no language header exists', async () => {
+    const wrapper = await mountWithLanguageHeader(<MockComponent />);
+
+    expect(wrapper.find(MockComponent)).toContainReactText('en');
+  });
+});
+
+function mountWithLanguageHeader(
```
Comment (lemonmade) [inline packages/react-network/src/test/hooks.test.tsx]: If you need a custom mount, use `createMount` instead

File: packages/react-network/src/test/hooks.test.tsx (hunk 2 above)
Comment (cartogram) [reply packages/react-network/src/test/hooks.test.tsx]: Yes you are correct. I am going to install a module to do the actual parsing here.

**Pull request merged**